pym/bob/input.py

```diff
@@ -593,27 +593,33 @@ def status(self, workspacePath, dir, verbose=0):
 
         status = ""
         longStatus = ""
-        if self.__commit:
-            output = self.callGit(workspacePath, 'rev-parse', 'HEAD').rstrip()
-            if output != self.__commit:
-                status += "S"
-                longStatus +=  colorize("   > commitId: configured: {}  actual: {}\n".format(self.__commit, output), "33")
-        elif self.__tag:
-            output = self.callGit(workspacePath, 'describe', '--tags', '--always').rstrip()
-            if output != self.__tag:
-                status += "S"
-                longStatus += colorize("    > tag: configured: {} actual: {}\n".format(self.__tag, output), "33")
-        elif self.__branch:
-            output = self.callGit(workspacePath, 'rev-parse', '--abbrev-ref', 'HEAD').rstrip()
-            if output != self.__branch:
-                status += "S"
-                longStatus += colorize("    > branch: configured: {} actual: {}\n".format(self.__branch, output), "33")
-            else:
-                output = self.callGit(workspacePath, 'rev-list', 'origin/'+self.__branch+'..HEAD')
-                if len(output):
-                    status += "U"
-                    # do not print detailed status this point.
-                    # git log --branches --not --remotes --decorate will give the same informations.
+
+        output = self.callGit(workspacePath, 'ls-remote' ,'--get-url').rstrip()
```

Comment:
hmm... not sure but maybe it's a documentation issue? I one of these stable distros:
```
$ git --version
git version 1.9.1
 $ git ls-remote -?
usage: git ls-remote [--heads] [--tags]  [-u <exec> | --upload-pack <exec>]
                     [-q|--quiet] [--exit-code] [--get-url] [<repository> [<refs>...]]
$ git ls-remote --get-url 
https://github.com/rhubert/bob.git
```

Doing some archaeological research in git I found this behavior was added with https://github.com/git/git/commit/9c00de5a3135c8f7273668d4013c225d48d47861#diff-f6a8f8e2f3b2ef7d425d564cddebf989 and first tagged with git 1.7.2-rc0.